admin/class-admin.php

```diff
@@ -201,7 +162,7 @@ function register_settings_page() {
 			foreach ( $submenu_pages as $submenu_page ) {
 
 				// Add submenu page
-				$admin_page = add_submenu_page( $submenu_page[0], $submenu_page[2] . ' - ' . __( 'Yoast WordPress SEO:', 'wordpress-seo' ), $submenu_page[2], $submenu_page[3], $submenu_page[4], $submenu_page[5] );
+				$admin_page = add_submenu_page( $submenu_page[0], $submenu_page[2] . ' - ' . __( 'WP SEO by Yoast', 'wordpress-seo' ), $submenu_page[2], $submenu_page[3], $submenu_page[4], $submenu_page[5] );
```

Comment:
For consistency, you might want to consider also changing similar strings (about 12). I'd also prefer to use "WordPress" instead of "WP" but I'll leave that up to you.